File: app/src/main/java/com/example/subcreator/MainActivity.java
package com.example.subcreator;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView textView = new TextView(this);
        textView.setText("SubCreator App Created Successfully!");
        textView.setTextSize(20);
        setContentView(textView);
    }
}

File: app/src/main/java/com/example/subcreator/MainActivity.kt
package com.example.subcreator

import android.os.Bundle
import android.app.Activity
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val textView = TextView(this).apply {
            text = "SubCreator App Created Successfully!"
            textSize = 20f
        }
        setContentView(textView)
    }
}
